data/bactrian.txt

```diff
@@ -0,0 +1,389 @@
+			`	As you approach them, the man stops talking with someone and notices you. "Captain <last>! Have you located the Star Queen?"`
+			choice
+				`	"Yes, it is located on <destination>, but is heavily defended."`
+			
+			branch destroyed
+				has "Bactrian Unlock [0]: Pirates: done"
+			`	You mention the multiple heavy warships that you saw over the planet, and the fact that there are probably many more waiting on the planet.`
+				goto next
+			
+			label destroyed
+			`	You mention the multiple heavy warships that you took out over the planet, but that there may be many more waiting on the planet.`
+			
+			label next
+			`	After hearing this news, many of the faces in the group turn grim as they realize that they will need to fight to retrieve the Star Queen. `
+			`	"We will need to get a rescue fleet ready. Soldiers, ready your ships!" The group responds with a collective "Sir yes sir!" before running off to their ships.`
+			`	Before leaving for his own ship, the man walks up to you and hands you 100,000 credits.`
+			`	"That's for finding the Star Queen. Now, Captain <last>, would you be willing to help us? If the Star Queen is as heavily defended as you say it is then we're going to need every ship we can get."`
+			choice
+				`	"I'd be glad to."`
+				`	"I would rather let the professionals handle things. Good luck."`
+					decline
+			
+			`	"Thank you so much. We'll be bringing a number of warships, but just make sure that the D.S.S. Faraday Corvette makes it onto the planet. The landing crew will be on that ship. Just launch whenever you're ready to fight."`
+			`	The man starts talking into some communication device, giving orders to the fleet that will be joining you on the mission, then puts the device back on his wasit and looks to you. "Also, since we'll be going into battle together, I might as well introduce myself. I'm Deep Lieutenant Shane Paris. Good luck fighting."`
+				accept
+	
+	npc save escort accompany
+		government Republic
+		personality heroic escort
+		ship "Corvette (Speedy)" "D.S.S. Faraday"
+	
+	on visit
+		dialog `You have landed on <planet>, but you left the <npc> behind! Wait for it to enter the system before landing.`
+
+mission "Bactrian Unlock [1]: Pirates"
+	invisible
+	landing
+	to offer
+		has "Bactrian Unlock [0]: done"
+	to fail
+		or
+			has "Bactrian Unlock [3]: done"
+			has "Bactrian Unlock [3-alt]: offered"
+	npc kill
+		personality nemesis staying
+		system Arneb
+		government Pirate
+		ship "Firebird (Plasma)" "The Court Jester"
+		ship "Firebird (Plasma)" "The Knighted One"
+		ship "Headhunter (Particle)" "Cerberus"
+		ship "Headhunter (Particle)" "Charon"
+
+mission "Bactrian Unlock [1]: Escorts"
+	invisible
+	landing
+	to offer
+		has "Bactrian Unlock [0]: done"
+	to fail
+		or
+			has "Bactrian Unlock [3]: offered"
+			has "Bactrian Unlock [3-alt]: offered"
+	npc escort kill
```

Comment:
I only did a local search on the data files of my 0.9.6 install ;-)
But because of the outcome being the same as your github search (no matches) I questioned the first (of several occurances) in bactrian.txt file of @Amazinite